e2e_deployment_files/nestededge_bottomLayerBaseDeployment_long_haul_mqtt.template.json

```diff
@@ -512,7 +512,34 @@
                 "value": "<TrackingId>"
               },
               "TEST_START_DELAY": {
-                "value": "120s"
+                "value": "240s"
+              },
+              "INITIATE_TOPIC": {
+                "value": "initiate/1"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-generic-mqtt-tester:<Build.BuildNumber>-linux-<Architecture>",
+              "createOptions": "{\"HostConfig\":{\"LogConfig\":{\"Type\":\"json-file\",\"Config\":{\"max-size\":\"4m\",\"max-file\":\"<LogRotationMaxFile>\",\"compress\":\"true\"}}}}"
+            }
+          },
+          "genericMqttReceiver2": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "TEST_SCENARIO": {
+                "value": "Receive"
+              },
+              "TEST_START_DELAY": {
+                "value": "1s"
+              },
+              "TRACKING_ID": {
+                "value": "<TrackingId>"
+              },
+              "INITIATE_TOPIC": {
+                "value": "initiate/2"
               }
             },
```

Comment:
Now the custom mqtt tests will have the following configuration:
L4 level: Initiator sending to L3 / Receiver receiving from L3
L3 level: Initiator sending to L4 / Receiver receiving from L4

The initiate topic defines what topic the test module will either subscribe or publish too.